### PR TITLE
Loki Dockerfile

### PR DESCRIPTION
**Pull request by tSudharshan**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:latest
+
+RUN yum update -y
+RUN yum install -y unzip
+LABEL ORG="Armedia LLC" \
```
Comment (david-oc-miller) [inline Dockerfile]: We need to use `ArkCase LLC` from now on; no longer `Armedia LLC`.

File: loki-config.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  grpc_listen_port: 9096
+
+common:
+  path_prefix: /loki/data
+  storage:
+    filesystem:
+      chunks_directory: /loki/data/chunks
+      rules_directory: /loki/data/rules
+  replication_factor: 1
+  ring:
```
Comment (david-oc-miller) [inline loki-config.yaml:14]: With this configuration, will Loki's own logs go through standard output only?  Or will Loki send its own log output to a folder within the configuration?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+#LOKI
+ARG LOKI_VERSION="2.4.2"
+
+ENV LOKI_USERID=2000 \
+    LOKI_GROUPID=2020 \
+    LOKI_GROUPNAME=loki \
+    LOKI_USER=loki \
+    LOKI_PORT=3100 \
+    LOKI_URL="https://github.com/grafana/loki/releases/download/v${LOKI_VERSION}/loki-linux-amd64.zip"
```
Comment (david-oc-miller) [inline Dockerfile:19]: Does Loki need to be configured to listen the log stream from each other container?  Or does it automatically discover the other containers and index them automatically?  Or does this configuration go into the orchestration layer (docker-compose, helm, etc) and doesn't belong in the image itself at all?

Comment (naveen-armedia) [reply Dockerfile:19]: @david-oc-miller Loki is a Log aggregation engine.  Each log stream from the container is indexed by its labels. And _**Promtail**_ will take care of sending logs to Loki from the running containers.

File: Dockerfile (hunk 1 above)
Comment (tSudharshan) [reply Dockerfile]: updated that

File: loki-config.yaml (hunk 2 above)
Comment (david-oc-miller) [reply loki-config.yaml:14]: @tSudharshan , where will Loki's own logs go?  Will they also be collected by Promtail, with no further configuration needed here?

Comment (tSudharshan) [reply loki-config.yaml:14]: yes promtail will collect loki's own logs as well

**Review by david-oc-miller — APPROVED** [review]

**Pull request merged**